arcgis-ios-sdk-samples/Layers/ArcGIS vector tiled layer (custom style)/VectorTileCustomStyle.storyboard

```diff
@@ -299,6 +301,63 @@
+                                                    <constraints>
+                                                        <constraint firstAttribute="trailing" secondItem="576-sA-6t9" secondAttribute="trailing" id="6p3-yZ-Gqu"/>
+                                                        <constraint firstAttribute="width" constant="40" id="COi-aI-nb5"/>
+                                                        <constraint firstAttribute="bottom" secondItem="u0W-73-arK" secondAttribute="bottom" id="KTo-Ad-wIJ"/>
+                                                        <constraint firstItem="u0W-73-arK" firstAttribute="top" secondItem="Eva-Qw-W63" secondAttribute="top" id="OIn-ZF-iTY"/>
+                                                        <constraint firstAttribute="bottom" secondItem="576-sA-6t9" secondAttribute="bottom" id="VF4-2J-aoy"/>
+                                                        <constraint firstItem="576-sA-6t9" firstAttribute="top" secondItem="Eva-Qw-W63" secondAttribute="top" id="kja-EU-nfo"/>
+                                                        <constraint firstItem="u0W-73-arK" firstAttribute="leading" secondItem="Eva-Qw-W63" secondAttribute="leading" id="oI0-GA-VbD"/>
+                                                        <constraint firstItem="576-sA-6t9" firstAttribute="leading" secondItem="u0W-73-arK" secondAttribute="trailing" id="tUF-BJ-aIw"/>
+                                                        <constraint firstAttribute="height" constant="30" id="uYg-UN-3d3"/>
+                                                    </constraints>
+                                                    <userDefinedRuntimeAttributes>
+                                                        <userDefinedRuntimeAttribute type="number" keyPath="layer.cornerRadius">
+                                                            <integer key="value" value="5"/>
+                                                        </userDefinedRuntimeAttribute>
+                                                    </userDefinedRuntimeAttributes>
+                                                </view>
+                                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Offlilne custom style" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontForContentSizeCategory="YES" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="LgN-yo-i5R">
+                                                    <rect key="frame" x="68" y="12" width="338" height="20"/>
+                                                    <fontDescription key="fontDescription" style="UICTFontTextStyleBody"/>
+                                                    <color key="textColor" red="0.0" green="0.0" blue="0.0" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
```

Comment:
What color issues are you referring to and how can I fix them?

Reply:
It would be great if you can check every color with `colorSpace="custom"` in the storyboard source code, and fix any that hasn't to be custom color.

But it is also OK to only fix the label colors for the cells (change them to default), so that the text are white in darkmode.

Reply:
Should I delete `colorSpace="custom"` altogether or should I set it to something else?

Reply:
For each custom color in the storyboard, find which view/property it belongs to, and change it to default color if it applies.

For example, for each static cell, when you tap on the label, it will show its label color is "custom". Reset it to default and in the storyboard it won't have the `colorSpace="custom"` anymore.

Reply:
There are other colors like cell background, tableview background, etc. that was set to custom due to legacy issues. It would be great if we can fix them whenever we see them.

Reply:
Colors look great now!